src/Microsoft.OpenApi/Models/OpenApiDocument.cs

```diff
@@ -83,7 +83,11 @@ public OpenApiDocument(OpenApiDocument document)
             Workspace = document?.Workspace != null ? new(document?.Workspace) : null;
             Info = document?.Info != null ? new(document?.Info) : null;
             Servers = document?.Servers != null ? new List<OpenApiServer>(document.Servers) : null;
-            Paths = document?.Paths != null ? new(document?.Paths) : null;
+            if (document?.Paths != null)
+            {
+                Paths = new();
```

Comment:
```suggestion
```

Reply:
Thanks. I've simplified it to: `Paths = document?.Paths`